bootloader/src/pyi_win32_utils.c

```diff
@@ -435,6 +435,55 @@ pyi_win32_utf8_to_mbs(char * dst, const char * src, size_t max)
     }
 }
 
+
+/* Retrieve the SID of the current user.
+ *  Returns SID string on success, NULL on failure.
```

Comment:
Please also explain that we need this code since wine versionX.Y does not support S-1-3-4 (directory owner).

This gives us a chance to remove the code when wine supports this SID.

Reply:
OK, done.